packages/datadog-plugin-mongodb-core/test/mongodb.spec.js

```diff
@@ -112,6 +113,165 @@ describe('Plugin', () => {
             collection.insertOne({ a: 1 }, {}, () => {})
           })
 
+          it('should have the statement tag when doing a single delete operation', async () => {
+            collection.deleteOne({ a: 1 }, {}, () => {})
+
+            return agent.assertSomeTraces(traces => {
+              const span = traces[0][0]
+              const query = '{"a":1}'
+              const resource = usesDelete ? 'delete' : 'remove'
+
+              expect(span).to.have.property('resource', resource + ` test.${collectionName}`)
+              expect(span.meta).to.have.property('mongodb.query', query)
+            })
+          })
+
+          it('should have the statement tag when doing a single deleteMany operation', async () => {
+            //  deleteMany and delete run the same command under the hood, they should have the same output
+            collection.deleteMany({ a: 1 }, {}, () => {})
+
+            return agent.assertSomeTraces(traces => {
+              const span = traces[0][0]
+              const query = '{"a":1}'
+              const resource = usesDelete ? 'delete' : 'remove'
+
+              expect(span).to.have.property('resource', resource + ` test.${collectionName}`)
+              expect(span.meta).to.have.property('mongodb.query', query)
+            })
```

Comment:
Could we already use our newer helper for these? :)

```suggestion
            const resource = usesDelete ? 'delete' : 'remove'

            return agent.assertFirstTraceSpan({
              resource: resource + ` test.${collectionName}`,
              span: {
                'mongodb.query': '{"a":1}',
              }
            })
```

The same for the other cases.

Reply:
It does seem like an appropriate change, on it!